src/components/articles.tsx

```diff
@@ -0,0 +1,16 @@
+import NavBar from './subnavigation/navbar';
+import React from 'react';
+import { PageWrapper } from '../styled/pages';
+
+const ArticlesComponent: React.FunctionComponent = () => {
+    return (
+        <>
+            <NavBar />
+            <PageWrapper>
```

Comment:
If `NavBar` is going to be included in every page, I think it would make more sense to put it in `components/layout.tsx` below the consistent nav to keep more in line with Next.js conventions and avoid repetition.

Reply:
Yep! good call @mcolella14